app/views/initiatives/index.html.erb

```diff
@@ -1,8 +1,10 @@
 <div class="wrapper">
   <div class="w-container w-row">
-    <h2 class="w-col w-col-7">Initiatives</h2>
+    <h2 class="w-col w-col-7"><%= "#{@team_member.fullname}'s Initiatives" %></h2>
     <div class="w-col w-col-5">
-      <%= link_to "New Initiative", new_initiative_path, class: 'btn-primary' %>
+      <% if !current_user.team_members.pluck(:id).include?(params[:user_id].to_i) %>
```

Comment:
What does this logic represent? How could you make it so it was more obvious and not a big long thing in the view?

Reply:
Resolved!